server/src/repositories/Users/UserRepository.ts

```diff
@@ -41,8 +41,19 @@ export async function createUser(user: {
   return await getUserByID(newID);
 }
 
-export function updateUser(args: any) {
-  throw new Error("Method not implemented.");
+export async function updateStudentProfile(args: {
```

Comment:
why single parameter? Seems fine just curious

Reply:
Any time I have several arguments of the same type I prefer to go with an argument object so I don't have to memorize the argument order lol